core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -287,6 +292,36 @@ public synchronized InetSocketAddress getAddress() throws UnavailableException {
    * @return the return value of the RPC call
    */
   protected synchronized <V> V retryRPC(RpcCallable<V> rpc) throws AlluxioStatusException {
+    return retryRPCInternal(rpc, () -> null);
```

Comment:
do you think it will make a difference if we pass null here and make a null check in `retryRPCInternal` for performance reason? Otherwise, I am cool with the current way.

Reply:
I would prefer to keep it this way and change to a null check if we notice performance issues.